README.adoc

```diff
@@ -36,29 +37,161 @@ include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/
 
 
 [[initial]]
-== Create a resource controller
+== Add required dependencies
+
+To use the Spring Integration GCP Pub/Sub channel adapters, you'll need to pull in the
+`spring-integration-gcp` dependency. We also strongly recommend the Spring Cloud GCP Pub/Sub Boot
+starter, `spring-cloud-gcp-starters-pubsub`, to deal with all the object instantiation and auto
+configuration.
+
+At the time of writing, the Spring Cloud GCP libraries are still in Alpha stage. As such, they are
+only published to Pivotal's Maven snapshot repository.
+
+To pull in these dependencies, add the following to your `pom.xml` file if you're using Maven:
+
+```
+<dependencies>
+    ...
+    <dependency>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-cloud-gcp-starter-pubsub</artifactId>
+        <version>1.0.0.BUILD-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-integration-gcp</artifactId>
+        <version>1.0.0.BUILD-SNAPSHOT</version>
+    </dependency>
+    ...
+</dependencies>
+
+
+
+<repositories>
+    ...
+    <repository>
+        <id>repository.spring.snapshot</id>
+        <name>Spring Snapshot Repository</name>
+        <url>http://repo.spring.io/snapshot</url>
+    </repository>
+    ...
+</repositories>
+```
+
+Or, if you're using Gradle:
+
+```
+repositories {
+    ...
+    maven {
+        url "http://repo.spring.io/snapshot"
+    }
+    ...
+}
+
+dependencies {
+    ...
+    compile("org.springframework.cloud:spring-cloud-gcp-starter-pubsub:1.0.0.BUILD-SNAPSHOT")
+    compile("org.springframework.cloud:spring-integration-gcp:1.0.0.BUILD-SNAPSHOT")
+    ...
+}
+```
+
+== Create an inbound channel adapter
+
+An inbound channel adapter listens to messages from a Google Cloud Pub/Sub subscription and sends
+them to a Spring channel in an application.
+
+Instantiating an inbound channel adapter requires a subscriber factory and the name of an existing
+subscription. The Spring Cloud GCP Pub/Sub Boot starter provides a pre-configured subscriber factory
+which you can simply inject as a method argument.
+
+`src/main/java/hello/PubsubApplication.java`
```

Comment:
Interesting! I would just add two more thoughts to reopen for a little bit that debate - even though it seems decided: 

1. based on the [Java Style Guide Camel Case section](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case), as [Pub/Sub](https://cloud.google.com/pubsub) is "hyphenated" (slashed?) - I would argue that it's two words, hence PubSub is the correct spelling. 

2. In the PubsubMessage proto file referenced, there is inconsistency as the [c# namespace is spelled PubSub](https://github.com/googleapis/googleapis/blob/cc6f9332e648a05b8504cd7b3c0cee3dd58648b2/google/pubsub/v1/pubsub.proto#L26)